src/explore-education-statistics-admin/src/components/editable/utils/getInvalidContent.ts

```diff
@@ -110,13 +102,64 @@ export default function getInvalidContent(
       return;
     }
 
-    // exclude glossary links
-    if (
-      link.data?.split(' ').length === 1 &&
-      !link.attributes?.linkHref?.includes('/glossary')
-    ) {
+    const badLinkTextWords = [
+      'click',
+      'csv',
+      'continue',
+      'dashboard',
+      'document',
+      'download',
+      'file',
+      'form',
+      'guidance',
+      'here',
+      'info',
+      'information',
+      'jpeg',
+      'jpg',
+      'learn',
+      'link',
+      'more',
+      'next',
+      'page',
+      'pdf',
+      'previous',
+      'read',
+      'site',
+      'svg',
+      'this',
+      'web',
+      'webpage',
+      'website',
+      'word',
+      'xslx',
+      'click here',
+      'click this link',
+      'download csv',
+      'download document',
+      'download file',
+      'download here',
+      'download jpg',
+      'download jpeg',
+      'download pdf',
+      'download png',
+      'download svg',
+      'download word',
+      'download xslx',
+      'further information',
+      'go here',
+      'learn more',
+      'link to',
+      'read more',
+      'this page',
+      'visit this',
+      'web page',
+      'web site',
+    ];
+
+    if (badLinkTextWords.includes(link.data?.toLowerCase()!)) {
```

Comment:
Unless this is already accounted for, we may also want to `trim` the link text to avoid allowing [ click here ]() being valid (with a leading or trailing space)

Reply:
Good catch 👍